nextcloud/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "2"
 services:
   nextcloud:
-    image: cusdeb/nextcloud:13RC4-armhf
+    image: cusdeb/nextcloud:13.0.1
```

Comment:
By default, MMB builds armhf images, so the version should look like `13.0.1-armhf`.